app/components/embed/companion_windows_component.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="sul-embed-container companion-window-component" id='sul-embed-object' hidden data-controller="companion-window <%= @stimulus_controller %>" data-media-tag-iiif-manifest-value="<%= iiif_v3_manifest_url %>" data-action="media-seek@window->media-tag#seek">
   <header aria-label="Embed main">
     <nav aria-label="Embed navigation">
-      <button data-action="click->companion-window#toggleLeft transcript#load" aria-label="Display sidebar" aria-controls="left-drawer">
+      <button data-action="click->companion-window#toggleLeft" aria-label="Display sidebar" aria-controls="left-drawer">
```

Comment:
Decoupled "transcript#load" from the companion window toggle, since the transcript will load when the first track is ready to parse